resources/logback.test.xml

```diff
@@ -0,0 +1,37 @@
+<configuration>
+    <property name="type" value="${ZIGGURAT_LOG_FORMAT:-text}" />
+
+    <if condition = 'property("type").equals("json")'>
+        <then>
+            <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+                <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+                    <layout class="cambium.logback.json.FlatJsonLayout">
+                        <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter">
+                            <prettyPrint>false</prettyPrint>
+                        </jsonFormatter>
+                        <!-- <context>api</context> -->
+                        <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSS'Z'</timestampFormat>
+                        <timestampFormatTimezoneId>UTC</timestampFormatTimezoneId>
```

Comment:
Does this make sense to hardcode the timezone?

Reply:
We should let it pick the timezone  and not hardcode

Reply:
we have removed it